esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/install_cold/bin/install_cold.sh

```diff
@@ -12,3 +15,16 @@ mkdir "${TARGET_DIR}"
 
 # Copy the environment file to the 'bin' directory.
 cp "${SOURCE_PATH}" "${TARGET_DIR}/${ENV_FILE}"
+
+# Remove esmvaltool and esmvalcore directories, if they exist.
+if [[ -d ${ESMVALTOOL_DIR} ]]; then
+    rm -rf "${ESMVALTOOL_DIR}"
+fi
+
+if [[ -d ${ESMVALCORE_DIR} ]]; then
+    rm -rf "${ESMVALCORE_DIR}"
+fi
+
+# Checkout main branch for ESMValTool and ESMValCore from github.
```

Comment:
Please include the comment I suggested previously, since it explains the use of the `-q` option.
```suggestion
# Checkout the specified branch for ESMValCore and ESMValTool. Use the
# quiet ('-q') option to prevent the progress status from being reported
# (which is done via done via 'stderr').
```